### PR TITLE
charts: update components to latest release

### PR DESCRIPTION
**Pull request by Haleygo**
1.update VictoriaMetrics components to v1.100.1
2.update VictoriaLogs to 0.5.2

### CODE REVIEW

**Review by Amper — APPROVED** [review]
LGTM

File: charts/victoria-metrics-single/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v1.99.0
+appVersion: v1.100.1
 description: Victoria Metrics Single version - high-performance, cost-effective and scalable TSDB, long-term remote storage for Prometheus
 name: victoria-metrics-single
 version: 0.9.17
```
Comment (zekker6) [inline charts/victoria-metrics-single/Chart.yaml:5]: Should chart version be updated?

File: charts/victoria-metrics-operator/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Helm Chart For Victoria Metrics Operator.
 
- ![Version: 0.29.5](https://img.shields.io/badge/Version-0.29.5-informational?style=flat-square)
+ ![Version: 0.29.6](https://img.shields.io/badge/Version-0.29.6-informational?style=flat-square)
```
Comment (zekker6) [inline charts/victoria-metrics-operator/README.md:3]: Seems like `Chart.yaml` wasn't commited

Comment (Haleygo) [reply charts/victoria-metrics-operator/README.md:3]: yeah, it was updated in https://github.com/VictoriaMetrics/helm-charts/commit/571d53572c09de33e774afc3761473868bcd0ed3, but doc didn't synced.

File: charts/victoria-metrics-single/Chart.yaml (hunk 1 above)
Comment (Haleygo) [reply charts/victoria-metrics-single/Chart.yaml]: yeah, thanks! 
updated

**Review by zekker6 — APPROVED** [review]

**Pull request merged**